.github/workflows/playwright.yaml

```diff
@@ -0,0 +1,49 @@
+name: Playwright Tests
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types:
+      [
+        opened,
+        edited,
+        synchronize,
+        reopened,
+        ready_for_review,
+        review_requested,
+      ]
+    branches:
+      - main
+jobs:
+  test:
+    if: github.event.pull_request.draft == false
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "20.x"
+          registry-url: "https://registry.npmjs.org"
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ github.ref }}
+      - uses: pnpm/action-setup@v4
+        with:
+          run_install: false
+      - name: Install and Build
+        run: |
+          pnpm install
+          pnpm build
```

Comment:
I copied this from the neighboring `tests.yaml`. I'm not sure if there's a better way to share this setup.